test.py

```diff
@@ -3,10 +3,14 @@
 
 app = Robyn()
 
+callCount = 0
```

Comment:
Why did you add a counter here??

Reply:
It was to demo the number of calls increasing and checking the GIL lock was acquiring the global parameters correctly. Was more for a personal testing scenario, could be reverted if needed

Reply:
No no. That's alright.